### PR TITLE
Fix reference entry

### PR DESCRIPTION
**Pull request by gfcas**
Fix a BibTeX entry for the online doc according to MathSciNet.

### CODE REVIEW

**Review by peterrum — APPROVED** [review]

File: doc/doxygen/references.bib [hunk 1]
```diff
@@ -489,10 +489,10 @@ @article{Engel2002
 
 @article{Brenner2009,
   author =    {S.C. Brenner and T. Gudi and L.-Y. Sung},
-  title =     {An a posteriori error estimator for a quadratic C0-interior penalty method for the biharmonic problem},
+  title =     {An a posteriori error estimator for a quadratic {$C^0$}-interior penalty method for the biharmonic problem},
   journal =   {IMA Journal of Numerical Analysis},
   publisher = {Oxford University Press ({OUP})},
-  year =      {2009},
+  year =      {2010},
```
[the remark this answers is not in the document]
Comment (gfcas) [reply doc/doxygen/references.bib:495]: A ``git grep 'Brenner2009' .`` did not found other places.

**Review by peterrum — APPROVED** [review]

**Comment by bangerth** [comment]
I should have checked this first. The entry is referenced in `examples/step-47/doc/results.dox`. Would you mind fixing the reference in a follow-up patch?

File: examples/step-47/doc/results.dox [hunk 2]
```diff
@@ -13,7 +13,7 @@ $Q_3$; and 5, 4, 3 for $Q_4$, respectively.
 
 From the literature, it is not immediately clear what
 the penalty parameter $\gamma$ should be. For example,
-@cite Brenner2009 state that it needs to be larger than one, and
+@cite Brenner2010 state that it needs to be larger than one, and
```
Comment (gfcas) [inline examples/step-47/doc/results.dox:16]: Do you mean this? This fix in the ``examples/step-47/doc/results.dox`` was already included in this PR.

**Comment by gfcas** [comment]
However, I have seen that a label ``Brenner2005`` in the intro.dox is not build correctly. Looking in the doc ``examples/setp-47/doc/intro.dox`` it looks good on first view for me (https://github.com/dealii/dealii/blob/2171362b37567dc6adc70ea83fe18d7d58f876f0/examples/step-47/doc/intro.dox#L229). Anything I do not see?

**Comment by bangerth** [comment]
I think this should probably have been a reference to @brenner2010 then. I'll fix this. Thanks for looking it up!

**Pull request merged**